source/energy/resource/renewable.rst

```diff
@@ -2,11 +2,11 @@
 
 Non-Biomass Renewable Resources
 ================================
-:numref:`tab-depl` shows the assumed total potentials of non-biomass renewable energy deployment (by resource type) in the MESSAGE model. In addition, the assumptions are compared 
-with technical potential estimates from the Global Energy Assessment (Rogner et al., 2012 :cite:`rogner_chapter_2012`). In this context, it is important to note that typical MESSAGE 
+:numref:`tab-depl` shows the assumed total potentials of non-biomass renewable energy deployment (by resource type) in the MESSAGE model. In addition, the technical potential estimates are based on different sources, 
+such as the U.S. National Renewable Energy Laboratory `database <https://www.nrel.gov/grid/data-tools.html>'_ as described in the Global Energy Assessment (Rogner et al., 2012 :cite:`rogner_chapter_2012`). 
```

Comment:
The link for the database can not be clicked and the reference appears with the 'cite' command but not as a number.

Reply:
Thanks for noticing this @GamzeUnlu95. Corrected. Can you please mention which reference is not working?